controls/radtimebar/getting-started.md

```diff
@@ -0,0 +1,284 @@
+
+The following example demonstrates how to populate a simple RadLinearSparkline with data and host it in RadTimeBar. Read more about the RadSparkline controls in its [help documentation]({%slug radsparkline_overview%}).
+
+#### __[XAML] Example 7: Hosting RadSparkline in RadTimeBar in XAML__
+{{region radtimebar-getting-started-9}}
+	<telerik:RadTimeBar PeriodStart="1-Jan-2019 12:00:00 AM" 
+						PeriodEnd="31-Dec-2019 11:59:59 PM"						
+						VisiblePeriodStart="14-March-2019 12:00:00 AM" 
+						VisiblePeriodEnd="11-Nov-2019 11:59:59 PM"
+						SelectionStart="1-Jun-2019 12:00:00 AM" 
+						SelectionEnd="25-Aug-2019 11:59:59 PM">
+		<telerik:RadTimeBar.Intervals>
+			<telerik:YearInterval />
+			<telerik:MonthInterval />
+			<telerik:WeekInterval />
+			<telerik:DayInterval />
+		</telerik:RadTimeBar.Intervals>
+		<telerik:RadTimeBar.Content>
+			<telerik:RadLinearSparkline x:Name="sparkline"/>
+		</telerik:RadTimeBar.Content>
+	</telerik:RadTimeBar>
+{{endregion}}
+
+#### __[C#] Example 8: Hosting RadSparkline in RadTimeBar in code__
+{{region radtimebar-getting-started-10}}
+	this.timeBar = new RadTimeBar();
+	timeBar.BeginInit();
+	timeBar.PeriodStart = new DateTime(2019, 1, 1, 12, 0, 0);
+	timeBar.PeriodEnd = new DateTime(2019, 12, 31, 23, 59, 59);
+	timeBar.VisiblePeriodStart = new DateTime(2019, 3, 14, 12, 0, 0);
+	timeBar.VisiblePeriodEnd = new DateTime(2019, 11, 11, 23, 59, 59);
+	timeBar.SelectionStart = new DateTime(2019, 6, 1, 12, 0, 0);
+	timeBar.SelectionEnd = new DateTime(2019, 8, 25, 23, 59, 59);
+	timeBar.Intervals.Add(new YearInterval());
+	timeBar.Intervals.Add(new MonthInterval());
+	timeBar.Intervals.Add(new WeekInterval());
+	timeBar.Intervals.Add(new DayInterval());
+	timeBar.EndInit();            
+	timeBar.Content = new RadLinearSparkline();
+{{endregion}}
+
+#### __[VB.NET] Example 8: Hosting RadSparkline in RadTimeBar in code__
+{{region radtimebar-getting-started-11}}	
+	Dim timeBar As RadTimeBar = New RadTimeBar()
+	timeBar.BeginInit()
+	timeBar.PeriodStart = New DateTime(2019, 1, 1, 12, 0, 0)
+	timeBar.PeriodEnd = New DateTime(2019, 12, 31, 23, 59, 59)
+	timeBar.VisiblePeriodStart = New DateTime(2019, 3, 14, 12, 0, 0)
+	timeBar.VisiblePeriodEnd = New DateTime(2019, 11, 11, 23, 59, 59)
+	timeBar.SelectionStart = New DateTime(2019, 6, 1, 12, 0, 0)
+	timeBar.SelectionEnd = New DateTime(2019, 8, 25, 23, 59, 59)
+	timeBar.Intervals.Add(New YearInterval())
+	timeBar.Intervals.Add(New MonthInterval())
+	timeBar.Intervals.Add(New WeekInterval())
+	timeBar.Intervals.Add(New DayInterval())
+	timeBar.EndInit()
+	timeBar.Content = New RadLinearSparkline()
+{{endregion}}
+
+#### __[C#] Example 9: Setting up the ItemsSource of the sparkline control__
+{{region radtimebar-getting-started-12}}
+	private static Random r = new Random();
+	
+	private void SetupSparkLineSource()
```

Comment:
The users may get confused where they should call the SetupSparkLineSource method. We should probably make this clearer.